db/seeds.rb

```diff
@@ -7,3 +7,175 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 require 'faker'
+
+puts 'Destroying clients'
+Client.destroy_all
+
+puts 'Destroying users'
+User.destroy_all
+
+puts 'Destroying organizations'
+Organization.destroy_all
+
+puts 'Destroying venues'
+Venue.destroy_all
+
+puts 'Destroying events'
+Event.destroy_all
+
+puts 'Destroying orders'
+Order.destroy_all
+
+puts 'Destroying event categories'
+EventCategory.destroy_all
+
+puts 'Destroying orders'
+Venue.destroy_all
+
+puts 'Destroying tickets'
+Ticket.destroy_all
+puts 'Done!'
+
+# CREATE RECORDS
+
+# Create organizations
+
+15.times do
+  organization = Faker::Sports::Football.unique.team
```

Comment:
would rename local variable "name" here
"organization" could be mistaken for the object